src/explore-education-statistics-common/src/components/form/rhf/RHFFormField.tsx

```diff
@@ -85,6 +88,7 @@ export default function RHFFormField<
         id={fieldId(name, customId)}
         name={name}
         inputRef={mergeRefs(fieldRef, inputRef)}
+        value={value}
```

Comment:
Doesn't this leave the component controlled?

Not sure this is great performance-wise as RHF typically updates the value of DOM elements directly via refs

Reply:
@ntsim as discussed, I've changed this to move getting the value to a new `RHFFormTextArea` component and created new `FormBaseTextArea` and `FormCharacterCount` components that are used by both the RHF and Formik versions.